src/server/worldserver/worldserver.conf.dist

```diff
@@ -2843,6 +2843,23 @@ AllowTrackBothResources = 0
 
 PlayerStart.AllReputation = 0
 
+#
+#    Weekend.XP.Days
```

Comment:
i would suggest to use a game_event to enable holiday xp rates

Reply:
Several id's?

Reply:
why?

Reply:
Oh nevermind, i didn't read it correctly.